README.md

```diff
@@ -105,6 +105,7 @@ A curated list of awesome HBase projects and resources.
 * [Omid](https://github.com/yahoo/omid) - Transactional support for HBase.
 * [Tephra](http://tephra.incubator.apache.org) - Globally consistent transactions on top of HBase.
 * [Themis](https://github.com/XiaoMi/themis) - Cross-row/cross-table transactions on HBase based on Google's Percolator.
+* [HBase-QoD](https://github.com/algarecu/hbase-0.94.8-qod) - Vector-field consistency for HBase fine-grained transactional inter-DC replication.
```

Comment:
Also please rename to `HBaseQoD`

Reply:
Nevermind, I see the paper uses `HBase-QoD` even though the repo uses `HBaseQoD`

Reply:
> Nevermind, I see the paper uses `HBase-QoD` even though the repo uses `HBaseQoD`

Cool, I have adjusted the name in my own repo for simplicity.